iothub/device/tests/HsmAuthentication/ModuleAuthenticationWithHsmTest.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information
+
+using System.Globalization;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Client.HsmAuthentication;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Microsoft.Azure.Devices.Client.Test.HsmAuthentication
+{
+    [TestClass]
+    public class ModuleAuthenticationWithHsmTest
+    {
+        string signature = "signature";
```

Comment:
>signature [](start = 15, length = 9)

From the coding standards:
"We use _camelCase for internal and private fields and use readonly where possible. Prefix internal and private instance fields with _, static fields with s_ and thread static fields with t_."

Please use underscore here